pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: linkrunner
 description: "Flutter Package for linkrunner, track every click, download and dropoff for your app links"
-version: 3.2.2
+version: 3.2.3
```

Comment:
It will be a minor version update, not a patch release, since we are adding new optional parameters.